addons/partner_autocomplete/static/src/xml/partner_autocomplete.xml

```diff
@@ -5,6 +5,7 @@
             <t t-elif="props.record.resModel !== 'res.partner' || props.record.data.company_type === 'company'">
                 <AutoComplete
                     value="props.value || ''"
+                    placeholder="props.placeholder || ''"
```

Comment:
Hello @angv-odoo 

Would it be possible to write a quick QUnit test that would cover this use case?

Reply:
Hi, I just added the Qunit test for it :+1: